app/posts/register-trainee-teachers/2023-01-31-asking-providers-to-sign-off-trainee-data-for-the-previous-academic-year.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Asking providers to sign off trainee data for the previous academic year
+description: We created a way for providers to confirm which trainees qualified, withdrew and deferred in the previous academic year.
+date: 2023-01-31
+---
```

Comment:
```suggestion
related:
  items:
    - text: Adding reports to Register to help users check their data
      href: /register-trainee-teachers/adding-reports-to-register/
    - text: Helping HEIs find trainees from a single year
      href: /register-trainee-teachers/identifying-trainees-from-a-single-year/
---
```

Reply:
I've added the reports page as a related link but not the one about finding trainees from a single year, as we link to it directly within the content.